RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -8,6 +8,14 @@ extension NonFungibleTokenDetails.State {
 		}
 	}
 
+	var title: Loadable<String?> {
```

Comment:
looks a bit weird with a `Loadable<T>` as computed property. But if it makes caller easier to impl, then sure...